packages/serverless-component/src/component.ts

```diff
@@ -42,9 +43,26 @@ class NextjsComponent extends Component {
     );
   }
 
+  readRoutesManifest(nextConfigPath: string): Promise<RoutesManifest> {
+    return readJSON(join(nextConfigPath, ".next/routes-manifest.json"));
+  }
+
+  pathPattern(pattern: string, routesManifest: RoutesManifest): string {
+    const basePath = routesManifest.basePath;
+    if (basePath.length === 0) {
+      return pattern;
+    }
+
+    const basePathPattern =
+      basePath.charAt(0) === "/" ? basePath.slice(1) : basePath;
```

Comment:
Out of curiosity why do we need this? Does next.js allow for base paths with and without `/`?

Reply:
It doesn't :) Fixed